### ISSUE TITLE
I'm not able to set automplete with a value different from 'nope'

### ISSUE THREAD

**Issue by Cameo8** [shentao/vue-multiselect#1434]
I've an issue on rayban.com, we need to change automplete attribute value in multiselect fro WCAG reasons.
SiteImprove reported that we can't use 'nope' value. 
I'm not able to give to autocomplete a different value, in my vue app. I can't specify it inside <multiselect>
How can I manage it ? Thanks
<img width="1792" alt="Schermata 2021-05-17 alle 11 50 10" src="https://user-images.githubusercontent.com/84318134/118469502-0bc3cc00-b706-11eb-8aad-ed510fc74504.png">

**Comment by mattelen** [comment]
Hi @Cameo8 

Thanks for reporting an issue, however I'm not able to replicate the issue. If you can replicate the issue in its own shareable separate environment (like in a jsfiddle:   https://jsfiddle.net/shentao/jqofkzxc/3296/) then we can look at fixing the issue. In the meantime, I'll close this issue

**Comment by Cameo8** [comment]
<img width="1792" alt="Schermata 2021-05-24 alle 11 57 32" src="https://user-images.githubusercontent.com/84318134/119331092-3c1be500-bc87-11eb-98c5-44c4ca62122c.png">
I repeat, in general seems not possible to change value nope for the attribute autocomplete, I'm wrong ? can you please clarify if there is a way to change this value ? 

Thanks

**Comment by rkomiyama** [comment]
For anyone who's trying to get this to work, version 2.1.0 works.

**Comment by Cameo8** [comment]
Hi @rkomiyama thanks for your feedback. 
We are using version 2.1.6 
<img width="1792" alt="Schermata 2021-06-04 alle 09 36 52" src="https://user-images.githubusercontent.com/84318134/120764404-663b9580-c518-11eb-86a5-7cae869758a3.png">

I've already tried to pass autocomplete attribute to the multiselect 
`<multiselect
            class="rb-country-overlay-modal__content"
            data-element-id="X_X_CountrySelector_DropDown"
            automplete="country"
            :data-description="`to` + destinationCountryIso + `_from` + currentCountryIso"
            :internal-search="false"
            label="label"
            :placeholder="typeLabel"
            :open-direction="openDirection"
            :options="filteredLocales"
            :show-labels="false"
            track-by="label"
            @search-change="findCoincidences"
            @select="changeCountry"
          >
          </multiselect>`
but it doesn't work. Have I to downgrade the version to 2.1.0 ? are you suggesting this ? 

Thanks

**Comment by rkomiyama** [comment]
@Cameo8 Yes, I am suggesting that you downgrade to 2.1.0 since it seems to set `autocomplete` to `off` by default.
